app/views/components/_date-filter.html.erb

```diff
@@ -7,6 +7,17 @@
 %>
 <% if key && name %>
   <div class="app-c-date-filter"<% if aria_controls_id %> data-module="enable-aria-controls"<% end %>>
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: name + " after"
+      },
+      name: key + "[from]",
+      id: key + "[from]",
+      value: from_value,
+      controls: aria_controls_id,
+      describedby: "date-help-text-" + key
+    } %>
+<% if false %>
```

Comment:
Has this been left in by mistake from some testing? There's also one further down (line 47)

Reply:
D'oh!!!